client/webserver/site/src/js/markets.js

```diff
@@ -214,9 +214,12 @@ export default class MarketsPage extends BasePage {
     this.market = {
       dex: dex,
       sid: marketID(base, quote), // A string market identifier used by the DEX.
+      // app.assets represents assets which are supported by dexc
       base: app.assets[base],
-      baseCfg: dex.assets[base],
       quote: app.assets[quote],
+      // dex.assets represents assets which are supported by dex server.
+      // dex assets don't have a wallet property.
```

Comment:
~~dex assets don't have a wallet property.~~

This is only one of many differences. Instead, describe their Go types `core.SupportedAsset` and `dex.Asset`, since the type definitions also serve as documentation of the JSON structure.